pkg/ebpf/events_processor.go

```diff
@@ -13,7 +12,6 @@ import (
 
 	"github.com/aquasecurity/tracee/pkg/events"
 	"github.com/aquasecurity/tracee/pkg/events/parse"
-	"github.com/aquasecurity/tracee/pkg/procinfo"
```

Comment:
Procinfo is actually the basis for future enrichments. I think we should keep it for now

Reply:
Yep, I had doubts about removing it.